parameter/CMakeLists.txt

```diff
@@ -103,8 +105,13 @@ add_library(parameter SHARED
     XmlFileIncluderElement.cpp
     XmlParameterSerializingContext.cpp)
 
+GENERATE_EXPORT_HEADER( parameter
```

Comment:
I don't mean to overload it but to write `some_command_name` instead of `SOME_COMMAND_NAME` consistently.